.github/workflows/test.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest]
-        python-version: ['3.8', '3.9']
+        python-version: ['3.8', '3.9', 'pypy-3.8-7.3.6rc2']
```

Comment:
Can we avoid pinning to a specific PyPy version/ i.e. would something like `pypy-3.8-*` work?

Reply:
'pypy-3.8' should work since it has now been released.